examples/applications/topics_extraction_with_nmf_lda.py

```diff
@@ -1,6 +1,7 @@
 """
 =======================================================================================
-Topic extraction with Non-negative Matrix Factorization and Latent Dirichlet Allocation
+Topic extraction with Non-negative Matrix Factorization and Latent 
+Dirichlet Allocation
 =======================================================================================
```

Comment:
Why change the title without changing the length of the bars around it? Have you checked that the rendering is still good with `cd doc && make html-noplot`?